### PR TITLE
EA-91: Deprecated MetadataUtil class in EMR API, 

### PR DESCRIPTION
**Pull request by mozzy11**
Ticket https://issues.openmrs.org/browse/EA-91
"EA-91: Remove MetadataUtil class from EMR API, removed the tests , and also removed the dependence on Metadasharing Module"

### CODE REVIEW

**Comment by mozzy11** [comment]
hi @djazayeri  , checkout this PR. i instead just deprecated the Util class in Emrapi and added an alias to the xtream Serializer and also added a test

File: api/src/main/java/org/openmrs/module/emrapi/utils/MetadataUtil.java [hunk 1]
```diff
@@ -48,6 +49,10 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+/**
+ * @deprecated . Should delegate to org/openmrs/module/metadatasharing/util/MetadataUtil.java
```
Comment (djazayeri) [inline api/src/main/java/org/openmrs/module/emrapi/utils/MetadataUtil.java]: Make this comment better. Need to say that:

* this method has moved
* you need to do two things:
    1. ensure you have at least version xyz of other module xyz
    1. change this to call method xyz of other module xyz
    1. change your metadata files to use xyz format
* more info at (link to JIRA ticket)

File: api/src/main/java/org/openmrs/module/emrapi/utils/MetadataUtil.java [hunk 2]
```diff
@@ -130,8 +135,10 @@ public static MetadataPackagesConfig getMetadataPackagesForModule(ClassLoader lo
 		try {
 			InputStream stream = loader.getResourceAsStream(packageFilePath);
 			String xml = IOUtils.toString(stream);
-			MetadataPackagesConfig config = Context.getSerializationService().getDefaultSerializer()
-					.deserialize(xml, MetadataPackagesConfig.class);
+			SimpleXStreamSerializer serializer = (SimpleXStreamSerializer) Context.getSerializationService().getDefaultSerializer();
```
Comment (djazayeri) [inline api/src/main/java/org/openmrs/module/emrapi/utils/MetadataUtil.java]: Don't make this change here. We only want to support two things:

1. User doesn't make any changes, things keep working as they were
2. User makes all the documented changes at once.

The only changes in this module should be adding deprecation notices, and instructions.

**Comment by djazayeri** [comment]
Please also add deprecation notices on MetadataPackagesConfig and MetadataPackageConfig

**Comment by mozzy11** [comment]
@djazayeri , i have worked on your comments. only added the deprecation and the notices

**Comment by gracepotma** [comment]
@dkayiwa can you review this?

**Pull request merged**